broken-site-reporting/README.md

```diff
@@ -52,5 +52,5 @@ for $testSet in test.json
     
     if $test.expectReportURLParams
         for $param in $test.expectReportURLParams
-            expect($url.contains($param.name + '=' + $param.value))
+            expect($url.matchesRegex(/[?&] + $param.name + '=' + $param.value + [&$]/))
```

Comment:
this is a bug that I made in extension's integration of the tests, doing simple `.contains` is not good enough, params should be checked with left and right boundary